### PR TITLE
Abb alt text

### LINKED ISSUE #1328: Images Alt text and labels - Adopt, Buy or Build

### PR DESCRIPTION
**Pull request by gcharest**
Fixes  #1328

### CODE REVIEW

File: _pages/en/strategy-build-vs-buy.md [hunk 1]
```diff
@@ -280,7 +290,8 @@ Based on the risk level, ESDC will adjust its decisions around acquiring softwar
 Risk levels are heavily influenced by the capability being enabled by software.
 The following diagram helps visualize how this risk level varies according with the type of capabilities:
 
-![A 3-layered circle showing that risk increases from exterior to interior circles, the most exterior one being Productivity, the middle one being Corporate or Shared, and the innermost being Core to Mandate](assets/images/strategy-build-buy-risk-circle.png){:height="50%" width="50%" style="display:block;margin-left:auto;margin-right:auto"}
+![A 3-layered circle with an arrow pointing from the external circle crossing all the way towards the internal circle.
+The external circle is called "Productivity", the middle circle is called "Corporate or Shared" and the internal circle is called "Core to Mandate"](assets/images/strategy-build-buy-risk-circle.png){:height="50%" width="50%" style="display:block;margin-left:auto;margin-right:auto"}
```
Comment (RachelMuston) [inline _pages/en/strategy-build-vs-buy.md:294]: maybe change to "A 3-layered circle. The external circle is called "Productivity", the middle circle is called "Corporate or Shared" and the internal circle is called "Core to Mandate". There is an arrow that starts at the external circle which points towards the inner circle and has the text "Risk increases" on it.

**Review by RachelMuston — COMMENTED** [review]
These are awesome descriptions! I just have some suggestions for minor edits to the alt text on the last image.

File: _pages/en/strategy-build-vs-buy.md (hunk 1 above)
Comment (gcharest) [reply _pages/en/strategy-build-vs-buy.md:294]: Thanks! I’ll make the change!

**Comment by gcharest** [comment]
@RachelMuston Do you feel comfortable with the proposed changes to the alt-text to approve the merge? Thanks! :+1:

**Review by RachelMuston — APPROVED** [review]

**Pull request merged**